datasetinsights/datasets/groceries_real.py

```diff
@@ -176,92 +174,6 @@ def _filepath(self, filename):
         """
         return os.path.join(self.root, self.version, filename)
 
-    @staticmethod
```

Comment:
Why are these deleted if the scope of this review covers only UnitySimulationDownloader and the interface design?

Reply:
Originally the PR was broader. I can revert this module if needed although it will no longer be called anywhere from the cli.

Reply:
I'll revert this